explorer/app/components/TableCreator/tableCreator.tsx

```diff
@@ -53,7 +53,8 @@ function isGridTrackMinMax(width: GridTrackSize): width is GridTrackMinMax {
 }
 
 const createCell = <T extends object>(config: ColumnConfig<T>) =>
-  function CellCreator({ row }: CellProps<T>): JSX.Element {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- We can't determine the cell type
+  function CellCreator({ row }: CellContext<T, any>): JSX.Element {
```

Comment:
Same as above, can you provide some context here as to why `any` is necessary here? Feel free to share the react-table upgrade guide on `CellProps` to `CellContext`, or even just the docs on `CellContext`.

Reply:
Checking the [type](https://github.com/TanStack/table/blob/c5cf4e70b957d987d02c30ea17df738beb284691/packages/table-core/src/types.ts#L201) and [the doc ](https://tanstack.com/table/v8/docs/api/core/column-def) the cell function receives the `CellContext` which now receives two generics, one to determine the data type of the table, and the second to determine the data type of the cell. Since we use the same render function for every cell, we can't determine the type beforehand.